addons/mail/static/src/core/common/record.js

```diff
@@ -449,6 +458,9 @@ export class Record {
     static preinsert(data) {
         return this.get(data) ?? this.new(data);
     }
+    static isCommand(data) {
+        return ["ADD", "DELETE"].includes(data?.[0]?.[0]);
+    }
```

Comment:
So commands are only supported as array of commands?
Can't just have `field =  ["ADD", ...]` for simplicity?

Reply:
I thought the same, but this is over-engineering (= more LOCs) for something we don't need yet.
And `[[cmd, data], [cmd, data]]` syntax is still needed (e.g. RTC does that in python formatters), so I prefer to do such improvements later